packages/app/src/specs/SpecsList.vue

```diff
@@ -464,4 +463,12 @@ function onRunAllSpecs (rowId: string) {
 .spec-list-container {
   height: calc(100% - 112px)
 }
+
+/**
+ * Can't put a group on the parent element as it has downstream effects on the styling of child components
```

Comment:
We should move from Windi to Tailwind - it's got named groups which solves this (and way better docs, and better maintained).